packages/bsky/src/data-plane/server/routes/quotes.ts

```diff
@@ -0,0 +1,32 @@
+import { ServiceImpl } from '@connectrpc/connect'
+import { Service } from '../../../proto/bsky_connect'
+import { Database } from '../db'
+import { paginate, TimeCidKeyset } from '../db/pagination'
+
+export default (db: Database): Partial<ServiceImpl<typeof Service>> => ({
+  async getQuotesBySubject(req) {
+    const { subject, cursor, limit } = req
+    const { ref } = db.db.dynamic
+
+    if (!subject?.uri) return { uris: [] }
+
+    let builder = db.db
+      .selectFrom('quote')
+      .where('quote.subject', '=', subject.uri)
+      .select(['quote.uri', 'quote.cid', 'quote.sortAt'])
+
+    const keyset = new TimeCidKeyset(ref('quote.sortAt'), ref('quote.cid'))
+    builder = paginate(builder, {
+      limit,
+      cursor,
+      keyset,
+    })
```

Comment:
We'll want an index on `quote` based on this access pattern 👍

Reply:
Added but do please check, not quite certain if this is what we want.